src/plugins/integrations/AWSIntegration.ts

```diff
@@ -289,7 +296,6 @@ class AWSIntegration implements Integration {
                   FirehoseRequestTypes[operationName] : 'READ',
```

Comment:
we might use empty operation type instead of `READ` if the operation is not known

Reply:
Done